stem/control.py

```diff
@@ -1237,8 +1237,12 @@ def get_info(self, params, default = UNDEFINED, get_bytes = False):
 
       if is_multiple:
         return reply
-      else:
+
+      try:
         return list(reply.values())[0]
```

Comment:
The code is mostly indented with 2 spaces, which is near unreadable (imo). PEP-8 states 4 spaces.  I'm assuming you inherited this.

Reply:
yeah, they have their reasons.